README.md

```diff
@@ -47,8 +62,6 @@ npm run build
 
 ### Firebase functions
 
-**Note:** Before you run the following steps, you will need to ensure you have access to the Firebase project you want to use. This access can be set up through the Firebase web UI.
-
```

Comment:
We probably want to keep this note here in case people jump straight to this section, rather than following through the readme entirely from start to finish.

Reply:
How about on the original ordering, a link to and fro? Too complicated?

Reply:
The issue I had following the README linearly was it told me to run `npm start` before I had Firebase account or even the required variables set, so I was greeted with an error:
```
  Installations: Missing App configuration value: "projectId" (installations/missing-app-config-values).
  getMissingValueError
  C:/cloned/sparkletown/src/helpers/extract-app-config.ts:54
  ```